src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb

```diff
@@ -0,0 +1,52 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Globalization
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis.Completion
+Imports Microsoft.CodeAnalysis.CSharp.Completion
+Imports Microsoft.CodeAnalysis.CSharp.Completion.Providers
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Text
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
+    Public Class CompletionServiceTests
+        <Fact>
+        Public Async Function TestComplemtionDoesNotCrashWhenSyntaxTreeNotPresent() As Task
+            Dim workspaceDefinition =
+            <Workspace>
+                <Project Language="NoCompilation" AssemblyName="TestAssembly" CommonReferencesPortable="true">
+                    <Document>
+                        var x = {}; // e.g., TypeScript code or anything else that doesn't support compilations
+                    </Document>
+                </Project>
+            </Workspace>
+            Using workspace = Await TestWorkspace.CreateAsync(workspaceDefinition)
+                Dim document = workspace.CurrentSolution.Projects.First.Documents.First
+                Dim completionService As AbstractCompletionService = New CSharpCompletionService()
```

Comment:
Seems odd this is saying CSharpCompletionService when we're not on a CSharp document.